SwiftUI/Model/SearchOperation/SearchOperation.swift

```diff
@@ -16,47 +16,38 @@
 import Defaults
 
 extension SearchOperation {
-    var results: [SearchResult] { get { __results.array as? [SearchResult] ?? [] } }
```

Comment:
This is only syntax change, we do not need to add `get`, it is going to be the same without it.